pair.go

```diff
@@ -27,6 +27,10 @@ type PairingSecret struct {
 	sync.Mutex
 }
 
+type PairingOptions struct{
+	WorkstationName string `json:"name"`
```

Comment:
For optional JSON fields, I prefer `*string` so that you can always distinguish between `""` and not-present.